libmultilabel/linear/linear.py

```diff
@@ -138,7 +135,7 @@ def thresholding_one_label(y: np.ndarray,
 
     nr_fold = 3
 
-    l = y.shape[0]
+    l = y.shape[0]  # number of instances
```

Comment:
You should rename variable if you find the need to add such a comment.
That said, `l` has been the symbol for number of instances in our papers for a while now, which is why it's used without explanation.